File: 20220907/모범답안/20220907_04.js/cardGame.py
def solution(A):
    A.sort()
    answer = 0
    length = len(A)
    while length > 0:
        if length % 2 == 1:
            answer += (length * A[-1])
            A.remove(A[-1])
        else:
            answer += (length * A[0] * -1)
            A.remove(A[0])
        length -= 1

    return answer

File: 20220907/모범답안/20220907_04.js/cardGame.js
function solution(A){
    var answer = 0;
    var length = A.length;
  
    A.sort();
  
    while(length > 0){
      if(length % 2 == 1){
        answer += (length * A[A.length - 1]);
        A.pop();
      }else{
        answer += (length * A[0] * -1);
        A.shift();
      }
  
      length -= 1;
    }
  
    return answer;
}  